<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataInvaildView</class>
 <widget class="QDialog" name="DataInvaildView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Data_InvalidView_lbTestMsg">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Data_InvalidView_lbTestName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Data_InvalidView_lbSerial">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Data_InvalidView_btBack">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="Data_InvalidView_lbIco">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Data_InvalidView_lbMessage">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Data_InvalidView_btDelete">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <zorder>Data_InvalidView_lbTestMsg</zorder>
  <zorder>Data_InvalidView_lbTestName</zorder>
  <zorder>Data_InvalidView_lbSerial</zorder>
  <zorder>Data_InvalidView_lbIco</zorder>
  <zorder>Data_InvalidView_lbMessage</zorder>
  <zorder>Data_InvalidView_btBack</zorder>
  <zorder>Data_InvalidView_btDelete</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
